### DISCUSSION TITLE
Is it possible to send notification when a related entity added / changed/ removed for a tracked entity?

### DISCUSSION THREAD

**Discussion by davinhun** [YetiForceCompany/YetiForceCRM#16884] [How to]
Hello

Let's say a user track a Partner. If something changed in that user, for example name changed, a notification will appear. That's fine.
But if a comment added to that partner or a documentation uploaded to this partner, no new notification appears.
If it possible to track this kind of modification also?

**Comment by mariuszkrzaczkowski** [comment]
have you seen this tracking mechanism?
![image](https://user-images.githubusercontent.com/10198654/218392037-096ac7f0-9f4b-4ac0-9f11-1e2910f0d390.png)
